core/services/job/orm.go

```diff
@@ -385,7 +399,7 @@ type DRSpecConfig interface {
 }
 
 func LoadEnvConfigVarsVRF(cfg DRSpecConfig, vrfs VRFSpec) *VRFSpec {
-	// Take the larger of the global vs specific.
+	// Take the largest of the global vs specific.
```

Comment:
That wasn't a typo, its actually correct to use "larger" when comparing only two objects